docker/edask/environment.yaml

```diff
@@ -1,198 +1,221 @@
-prefix: /opt/conda/envs/edask
+  - absl-py=0.6.1=py36_1000
+  - asn1crypto=0.24.0=py36_1003
+  - astor=0.7.1=py_0
+  - atomicwrites=1.2.1=py_0
+  - attrs=18.2.0=py_0
+  - bcrypt=3.1.4=py36h470a237_0
+  - blas=1.1=openblas
+  - bokeh=1.0.1=py36_1000
+  - bottleneck=1.2.1=py36h7eb728f_1
+  - bzip2=1.0.6=h470a237_2
+  - c-ares=1.15.0=h470a237_1
+  - ca-certificates=2018.10.15=ha4d7672_0
+  - cairo=1.14.12=h276e583_5
+  - cartopy=0.16.0=py36h81b52dc_2
+  - cdat_info=8.0=py36_0
+  - cdms2=3.1.0=py36h6091dcd_0
+  - cdtime=3.1.0=py36h6091dcd_1
+  - cdutil=8.0=py36_1
+  - certifi=2018.10.15=py36_1000
+  - cffi=1.11.5=py36h5e8e0c9_1
+  - cftime=1.0.2.1=py36h7eb728f_0
+  - chardet=3.0.4=py36_1003
+  - clapack=3.2.1=h470a237_1
+  - click=7.0=py_0
+  - cloudpickle=0.6.1=py_0
+  - conda=4.5.11=py36_1000
+  - cryptography=2.3.1=py36hdffb7b8_0
+  - cryptography-vectors=2.3.1=py36_1000
+  - curl=7.62.0=h74213dd_0
+  - cycler=0.10.0=py_1
+  - cytoolz=0.9.0.1=py36h470a237_1
+  - dask=0.20.2=py_0
+  - dask-core=0.20.2=py_0
+  - dbus=1.13.0=h3a4f0e9_0
+  - defusedxml=0.5.0=py_1
+  - distarray=2.12.2=py_1
+  - distributed=1.24.2=py36_1000
+  - eofs=1.3.0=py_1
+  - esmf=7.1.0r=h35eb876_3
+  - esmpy=7.1.0r=py36_1
+  - expat=2.2.5=hfc679d8_2
+  - fontconfig=2.13.1=h65d0f4c_0
+  - freetype=2.9.1=h6debe1e_4
+  - future=0.17.0=py36_1000
+  - g2clib=1.6.0=3
+  - gast=0.2.0=py_0
+  - genutil=8.1.np1.14=py36he59b558_0
+  - geos=3.6.2=hfc679d8_3
+  - gettext=0.19.8.1=h5e8e0c9_1
+  - glib=2.56.2=h464dc38_1
+  - graphite2=1.3.12=hfc679d8_1
+  - graphviz=2.38.0=h08bfae6_9
+  - grpcio=1.16.0=py36hd60e7a3_0
+  - gsl=2.2.1=blas_openblashddceaf2_6
+  - gst-plugins-base=1.12.5=hde13a9d_0
+  - gstreamer=1.12.5=h5856ed1_0
+  - h5py=2.8.0=py36h097b052_4
+  - harfbuzz=1.9.0=h04dbb29_1
+  - hdf4=4.2.13=h951d187_2
+  - hdf5=1.10.3=hc401514_2
+  - heapdict=1.0.0=py36_1000
+  - icu=58.2=hfc679d8_0
+  - idna=2.7=py36_1002
+  - jasper=1.900.1=hff1ad4c_5
+  - jinja2=2.10=py_1
+  - jpeg=9c=h470a237_1
+  - keras=2.2.4=py36_0
+  - keras-applications=1.0.4=py_1
+  - keras-preprocessing=1.0.2=py_1
+  - kiwisolver=1.0.1=py36h2d50403_2
+  - krb5=1.16.2=hbb41f41_0
+  - lapack=3.6.1=1
+  - libcdms=3.0.1=h9ac9557_2
+  - libcf=1.0.2=py36h426daee_5
+  - libcurl=7.62.0=hbdb9355_0
+  - libdrs=3.1.0=h6e3784b_0
+  - libdrs_f=3.1.0=h2cd7f18_0
+  - libgcc=7.2.0=h69d50b8_2
+  - libgfortran=3.0.0=1
+  - libgfortran-ng=7.2.0=hdf63c60_3
+  - libgpuarray=0.7.6=h470a237_3
+  - libiconv=1.15=h470a237_3
+  - libnetcdf=4.6.1=h9cd6fdc_11
+  - libpng=1.6.35=ha92aebf_2
+  - libprotobuf=3.6.1=hd28b015_0
+  - libsodium=1.0.16=h470a237_1
+  - libssh2=1.8.0=h5b517e9_2
+  - libtiff=4.0.9=he6b73bb_2
+  - libtool=2.4.6=h470a237_2
+  - libuuid=2.32.1=h470a237_2
+  - libxcb=1.13=h470a237_2
+  - libxml2=2.9.8=h422b904_5
+  - libxslt=1.1.32=h88dbc4e_2
+  - locket=0.2.0=py_2
+  - lxml=4.2.5=py36hc9114bc_0
+  - mako=1.0.7=py_1
+  - markdown=2.6.11=py_0
+  - markupsafe=1.1.0=py36h470a237_0
+  - matplotlib=3.0.2=py36h8a2030e_1
```

Comment:
do you need this?

Reply:
Tom has it as a dependency, but it might be possible to take it out. I'd have to figure what are the minimal required dependencies to run.